src/main/webapp/dev/js/common/questionRubric.es6

```diff
@@ -10,6 +10,110 @@ import {
     disallowNonNumericEntries,
 } from './ui.es6';
 
+function getRubricChoiceElem(questionNum, col) {
+    return $(`#rubricChoice-${questionNum}-${col}`);
+}
+
+function getRubricWeightElem(questionNum, col) {
+    return $(`#rubricWeight-${questionNum}-${col}`);
+}
+
+function getRubricDescElem(questionNum, row, col) {
+    return $(`#rubricDesc-${questionNum}-${row}-${col}`);
+}
+
+function swapRubricCol(questionNum, colIndex, isSwapLeft) {
```

Comment:
`function moveRubricCol(questionNum, colIndex, isMoveLeft)`

If you insist on not taking in two elements and swapping them, and instead would rather take in a single element that can be moved in the left or right direction, then what is being done here is not really swapping but moving.